core/src/main/java/com/google/android/fhir/search/MoreSearch.kt

```diff
@@ -70,7 +70,7 @@ fun Search.getQuery(): SearchQuery {
     SELECT a.serializedResource
     FROM ResourceEntity a
     $sortJoinStatement
-    WHERE a.resourceType = ?
+    WHERE a.resourceType collate nocase = ?
```

Comment:
Are you sure this is not breaking the query?

Also please use all caps for keywords.

```suggestion
    WHERE a.resourceType = ? COLLATE NOCASE
```

https://www.designcise.com/web/tutorial/how-to-do-case-insensitive-comparisons-in-sqlite

Reply:
Sure @jingtang10  will follow that.